aicssegmentation/tests/test_WorkflowStep.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+from aicssegmentation.structure_wrapper.WorkflowEngine import WorkflowStep
+from skimage import data
+import pytest
+
+
+class TestWorkflowEngine:
+    def setup_method(self):
+        caller = getattr(data, "astronaut")
+        image = caller()
+        self.fake_image = np.asarray(image)
+
+        self.test_dict = dict()
+        self.test_dict["name"] = "intensity_normalization"
```

Comment:
Since you're effectively creating a new WorkflowStep in each test, creating one in your setup is somewhat useless. Not a big deal but I think it makes the test file more confusing.